src/vm/dac/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
-
 include(${CLR_DIR}/dac.cmake)
+include(CMakeDetermineCXXCompiler)
```

Comment:
@janvorli, I found this approach ([here](https://android-review.googlesource.com/c/platform/ndk/+/326383/6/build/cmake/android.toolchain.cmake#573)) better than passing custom variables in case of cross toolchain as we did previously:

https://github.com/dotnet/coreclr/blob/8a13881ea1d274067ac03b317fdbd9bcc9369990/cross/toolchain.cmake#L38-L41

perhaps that can be cleaned up as it may also simplify/unify cross toolchain configuration.